tests/system/small/test_session.py

```diff
@@ -1036,6 +1036,26 @@ def test_read_csv_local_w_usecols(session, scalars_pandas_df_index, engine):
         assert len(df.columns) == 1
 
 
+@pytest.mark.parametrize(
+    "engine",
+    [
+        pytest.param(
+            "bigquery",
+            id="bq_engine",
+            marks=pytest.mark.xfail(
+                raises=NotImplementedError,
```

Comment:
So it only available in default engine? We should update the docs to be explicit. Also should update the doc that we support global paths(currently says local only).

Reply:
Good catch! Done.